### DISCUSSION TITLE
Receiving FSK signal lower RSSI than RF noise floor

### DISCUSSION THREAD

**Discussion by Sumya99** [jgromes/RadioLib#594] [Q&A] [answered]
Hello there,

I am sorry about this question is may not relevant to the library,

Since CubeSats use FSK modulation for the downlink it should be possible that receiving FSK signal from 400km or more away. What I am facing is RF noise floor for the receiver side is around **-90dBm** and receiving signal is lower than this. So, I think this is the reason transmittion is not working.  I calculated Free Space Loss which is -137dBm even my transmitter generates +30dBm /1U cubesat restriction/ it seems impossible. 

Please let me know how can I receive FSK signal under floor noise?

Thanks in advance!

<img width="538" alt="image" src="https://user-images.githubusercontent.com/48245515/197335959-ed5bf863-e49e-4f9c-8a29-4ffe132da447.png">

**Comment by jgromes** [comment, marked as answer]
Short answer - no, you cannot receive FSK signal that is below noise floor, at that point, the signal is lost. There are modulations which can overcome this limit (such as LoRa), but that's not the case for FSK.

Instead, you have to design your link so that at the end, your signal-to-noise ratio is sufficient. Since you mentioned CubeSats, this will usually involve a large tracking yagi antenna, which can add 13 - 15 dB gain at the receiving end. Combined with things like low-noise amplifiers, you can receive signals from 1500 km away (which is the actual distance to a CubeSat on 500 km orbit couple of degrees above the horizon).

Proper link budget is a bit more complicated than just getting the free-space path loss, you have to include polarization losses, environmental losses, noise temperatrue at the receiver (to calculate the real noise floor of your system) etc., there are loads of online resources.
